CHANGELOG.md

```diff
@@ -1,7 +1,31 @@
 # Changelog
 
 ## Version 0.39.0
-_In development_
+
+<!-- ([Full Changelog](https://github.com/unitaryfund/mitiq/compare/v0.38.0...v0.39.0)) -->
```

Comment:
There is actually a problem with this that @jordandsullivan spotted and reported in #2438.

Reply:
I like the solution you posted [here](https://github.com/unitaryfund/mitiq/issues/2438#issuecomment-2272108429). Implemented in https://github.com/unitaryfund/mitiq/pull/2457/commits/c3d514d24828ba7912c682573af5c06472ad39a3.